GettingStarted/src/Samples/Positioning/SGSPositioningVC.m

```diff
@@ -0,0 +1,143 @@
+//
+//  SGSPositioningVC.m
+//  GettingStarted
+//
+//  Created by Fernando Sanchez Vilas on 28/03/2019.
+//  Copyright © 2019 Situm Technologies S.L. All rights reserved.
+//
+
+#import "SGSPositioningVC.h"
+@import CoreLocation;
+
+@interface SGSPositioningVC () <SITLocationDelegate>
+
+@property (weak, nonatomic) IBOutlet UIButton *startPositioningButton;
+@property (weak, nonatomic) IBOutlet UILabel *statusLabel;
+@property (weak, nonatomic) IBOutlet UILabel *positionLabel;
+
+@property (nonatomic, strong) CLLocationManager *locationManager;
+@property (nonatomic) BOOL isPositioning;
+@property (nonatomic, strong) NSDictionary *locationStatesTranslator;
+
+@end
+
+// Request user permission strings
+static NSString *PermissionDeniedAlertTitle = @"Location Authorization Needed";
+static NSString *PermissionDeniedAlertBody = @"This app needs location authorization to work properly. Please go to settings and enable it.";
+static NSString *PermissionDeniedOk = @"Ok";
+
+@implementation SGSPositioningVC
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self customizeNavigationBarWithTitle:@"Positioning"];
+    self.isPositioning = NO;
+    self.locationStatesTranslator = @{@(0):@"Stopped",
```

Comment:
Pareceme correcto o uso do diccionario para manter as traduccións. O único que si cambiaria é os numeros polos valores do enum `SITLocationState`, que considero que deixaría un pouco máis claro o tema.

Reply:
Correcto, boa idea